molconfgen/output.py

```diff
@@ -0,0 +1,79 @@
+# Richard Richardson wrote this ;)
+# Function to add a box to the output of
+# sampler.generate_conformers
+
+import MDAnalysis as mda
+import MDAnalysis.transformations
+import numpy as np
+
+
+# Function for finding the largest radius in
+# the trajectory
+# this will come in handy later
+def largest_r(u):
```

Comment:
Add a doc string in numpy doc string format (instead of a comment). For example
```python
def largest_r(ag):
   """Find the largest radius to enclose `ag`.

   The function iterates over the whole trajectory associated with `ag`
   and returns the maximum radius.

   Arguments
   ---------
   ag : AtomGroup
      `ag` contains the molecule of interest. 

   Returns
   -------
   r : float
```
See https://github.com/Becksteinlab/mdpow-molconfgen/blob/16c6513a01ba4ec610a01a6419533d2f4abf3fc9/molconfgen/sampler.py#L24 for example.